docs/components/pricing-table.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: '`<PricingTable />`'
+description: Clerk's <PricingTable /> component displays a table of Plans and Features that users can subscribe to.
+---
+
+The `<PricingTable />` component displays a table of Plans and Features that users can subscribe to.
+
+## Properties
+
+<Properties>
+  - `appearance`
+  - <code>[Appearance](/docs/customization/overview) | undefined</code>
+
+  Optional object to style your components. Will only affect [Clerk components](/docs/components/overview) and not [Account Portal](/docs/account-portal/overview) pages.
+
+  ---
+
+  - `checkoutProps`
+  - `{ appearance: Appearance }`
+
+  Options for the checkout popup. Accepts the following properties:
+
+  - [`appearance`](/docs/customization/overview): an object used to style your components. Will only affect [Clerk components](/docs/components/overview) and not [Account Portal](/docs/account-portal/overview) pages.
+
+  ---
+
+  - `collapseFeatures`
+  - `boolean`
+
+  A boolean that indicates whether the features are collapsed. **Requires `layout` to be set to `'default'`**. Defaults to `false`.
+
+  ---
+
+  - `ctaPosition`
+  - `'top' | 'bottom'`
+
+  The placement of the CTA button. **Requires `layout` to be set to `'default'`**. Defaults to `'bottom'`.
+
+  ---
+
+  - `fallback?`
+  - `JSX`
+
+  An optional UI to show when the pricing table is loading.
+
+  ---
+
+  - `forOrganizations`
+  - `boolean`
+
+  A boolean that indicates whether the pricing table is for [organizations](/docs/organizations/overview). If `false`, the pricing table will display a list of plans and features that **users** can subscribe to. If `true`, the pricing table will display a list of plans and features that **organizations** can subscribe to. Defaults to `false`.
+</Properties>
+
+## Usage with frameworks
+
+The following example includes a basic implementation of the `<PricingTable />` component. You can use this as a starting point for your own implementation.
+
+<Tabs items={["Next.js", "React", "Astro", "React Router", "Tanstack React Start"]}>
```

Comment:
We're missing examples for Vue and Nuxt here. The component name is also different for these: https://github.com/clerk/javascript/pull/5769